runtime/native/src/test/kotlin/kotlinx/serialization/features/ContextAndPolymorphic.kt

```diff
@@ -0,0 +1,98 @@
+package kotlinx.serialization.features
```

Comment:
I don't think that this is a json test since it mainly tests context features. Json is used here just because this is a simple text format for which 'gold' data can be provided